tests/nonbonded/test_nonbonded_interaction_group.py

```diff
@@ -316,15 +315,14 @@ def test_nonbonded_interaction_group_consistency_allpairs_constant_shift(
     lambda_offset_idxs = rng.integers(-2, 3, size=(num_atoms,), dtype=np.int32)
 
     def ref_allpairs(conf, lamb):
-        return prepare_reference_nonbonded(
-            params=params,
+        return generic.Nonbonded(
             exclusion_idxs=np.array([], dtype=np.int32),
-            scales=np.zeros((0, 2), dtype=np.float64),
+            scale_factors=np.zeros((0, 2), dtype=np.float64),
             lambda_plane_idxs=lambda_plane_idxs,
             lambda_offset_idxs=lambda_offset_idxs,
             beta=beta,
             cutoff=cutoff,
-        )(conf, params, example_box, lamb)
+        ).to_reference()(conf, params, example_box, lamb)
```

Comment:
Personal style preference: more lines `nb = generic.Nonbonded(...); U_ref = nb.to_reference(); return U_ref(...)` or similar

Reply:
Agreed; applied in 962dcc790ce4ea4c9e5f574f06c40c26d4f09114